spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/RibbonClientConfigDefaults.java

```diff
@@ -0,0 +1,79 @@
+package org.springframework.cloud.netflix.ribbon;
+
+import java.util.concurrent.TimeUnit;
+
+public abstract class RibbonClientConfigDefaults {
```

Comment:
Are these defaults taken from the underlying Ribbon library?  What if they change, how will we know?

Reply:
Yes, they are taken from `DefaultClientConfigImpl` @ryanjbaxter - that is a good point, don't see a good way other than referring back to `DefaultClientConfigImpl`. I was trying to remove all references to this class in the codebase and decided to replicate even the defaults. But may be we can shadow the defaults in `RibbonClientConfigDefaults` by pointing them to `DefaultClientConfigImpl`, this way the only reference to this class remains in `RibbonClientConfigDefaults`. Still a POC though, I still feel that something is going to break with this approach that is not captured in tests

Reply:
Changed the default to shadow`DefaultClientConfigImpl` now @ryanjbaxter